src/graphql/helpers.ts

```diff
@@ -34,6 +34,15 @@ export function formatSpace(id, settings) {
   return space;
 }
 
+export function formatUser(user) {
+  const profile = jsonParse(user.profile ?? {});
```

Comment:
The first argument on jsonParse should be a string, now you are using empty object if profile is empty. You can do like this instead:
```suggestion
  const profile = jsonParse(user.profile, {});
```